modules/broadcast/src/backend/db.ts

```diff
@@ -149,21 +149,20 @@ export default class BroadcastDb {
     tz = padDigits(Math.abs(Number(tz)), 2)
     const relTime = moment(`${schedule['date_time']}${sign}${tz}`, 'YYYY-MM-DD HH:mmZ').toDate()
     const adjustedTime = this.knex.date.format(schedule['ts'] ? schedule['ts'] : relTime)
-    const whereClause = _.isNil(initialTz) ? "where attributes -> 'timezone' IS NULL" : "where attributes -> 'timezone' = :initialTz"
+    const whereClause = _.isNil(initialTz) ? "where attributes -> 'timezone' IS NULL" : `where attributes ->> 'timezone' = '${initialTz}'`
```

Comment:
This brings a possible SQL Injection if you add an sql statement in the `timezone` attribute of any user, since there isn't any validation of the input

Reply:
@dmytropanontko why are you writing manual SQL?

Reply:
@slvnperron please, look again, for some reasons not all my changes was pushed, sorry